frontend/src/components/funding/FundingAgreementsTab.vue

```diff
@@ -59,12 +60,43 @@ export default {
     ...mapState(useAuthStore, ['userInfo']),
   },
 
-  created() {
+  async created() {
+    this.BASE_FUNDING = 'Base Funding'
```

Comment:
This constant isn't used in the template, so it could just be defined like a regular js constant, before export default.